### ISSUE TITLE
Network device name during bootup

### ISSUE THREAD

**Issue by linusyong** [rancherio/os#171]
The issue as describe below:
* ISO file: rancheros.iso 0.3.0-rc2
* Environment: VirtualBox 4.3.12 r93733 Windows
* Network: bridge or NAT (both tested)

During first time boot up, `ifconfig` does not show any ethernet device.  `/var/log/messages` shows that ethernet device has been renamed using udev predictable name:
```
Apr  13 05:07:21 rancher kernel: [    5.633363] e1000 0000:00:03.0 enp0s3: renamed from eth0
```

`ifconfig enp0s3` shows the device is there and `dhcpcd enp0s3` gives the device IP.

**Comment by deniseschannon** [comment]
This is only related to v0.3.0-rc2:

For a temporary fix:

In the VM in VirtualBox, go to Network Settings -> Advanced Settings.

Change "Adapter Type" to "Paravirtualized Network (virtio-net)".

**Comment by deniseschannon** [comment]
@linusyong This is fixed in v0.3.0-rc2. Please try with that version and see if this issue is happening.

**Comment by deniseschannon** [comment]
Fixed in v0.3.0
